htmresearch/support/neural_correlations_utils.py

```diff
@@ -153,7 +153,7 @@ def accuracy(current, predicted):
   """  
   acc = 0
   if np.count_nonzero(predicted) > 0:
-    acc = float(np.dot(current, predicted))/float(np.count_nonzero(predicted))
+    acc = float(np.dot(current, predicted))/float(np.count_nonzero(current))
```

Comment:
this one takes into account multiple predictions. If presented with "ABX" and "ABY" the TM will predict "X" and "Y" after seeing "AB". If the next SDR seen by the TM is "X" then the accuracy will be 1.